### PR TITLE
Various mostly Covenant changes

### PR DESCRIPTION
**Pull request by caelaislinn**
See commit log. Ready to merge

### CODE REVIEW

File: maps/~mapsystem/maps_jobs.dm [hunk 1]
```diff
@@ -7,6 +7,12 @@
 	if(!istype(S) || !istype(J))
 		return TRUE
 
+	//check the job itself first, before checking the map
+	if(S.type in J.blacklisted_species)
+		return TRUE
```
Comment (bloxgate) [inline maps/~mapsystem/maps_jobs.dm:12]: I assume this is checking if the Species is good for the job?

If so, shouldn't it return false if that species is blacklisted for the job? A blacklist is supposed to be things that aren't allowed after all.

**Review by bloxgate — COMMENTED** [review]
Looks good for the most part. One question though.

Once that's resolved, if this passes Travis, it should be good to merge.

File: maps/~mapsystem/maps_jobs.dm (hunk 1 above)
Comment (caelaislinn) [reply maps/~mapsystem/maps_jobs.dm:12]: This proc name = "is_species_job_restricted" and returns true if the species can't use that job

Comment (bloxgate) [reply maps/~mapsystem/maps_jobs.dm:12]: Then isn't the rest of this wrong instead? Right below this you return FALSE if the species is **not** whitelisted for the job.

Comment (caelaislinn) [reply maps/~mapsystem/maps_jobs.dm:12]: "is_species_job_restricted" returns true if the species argument is restricted from the job argument

**Review by bloxgate — DISMISSED** [review]

**Comment by caelaislinn** [comment]
Travis failure is the exoplanet simple mob

**Review by BDpuffy420 — APPROVED** [review]

**Comment by caelaislinn** [comment]
Ready to merge post travis

**Pull request merged**